modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -74,6 +79,9 @@ namespace detail {
 
         // NB: Number of asyncrhonious infer requests
         size_t nireq;
+
+        cv::util::any context_config;
+        std::shared_ptr<InferenceEngine::RemoteContext> rctx;
```

Comment:
I don't see a method to configure this field - is it still required?

Reply:
There is no way we configure it here due to IE dependency - we just setup a config class. The configuration occurs later during `IEUnit` creation in the backend

Reply:
`context_config` does nothing by itself - it is used to configure `rctx`

Reply:
I mean do you want user to setup the context from outside and just pass it here?

Reply:
Remove from here. However, see import/load network API changes